addon/ActivityStreams.js

```diff
@@ -105,12 +104,8 @@ function ActivityStreams(metadataStore, options = {}) {
   );
   this._experimentProvider.init();
 
-  this._tabTracker = new TabTracker(
-    this.appURLs,
-    options.clientID,
-    this._memoized,
-    this._experimentProvider.experimentId
-  );
+  this._tabTracker = tabTracker;
+  this._tabTracker.init(this.appURLs, this._memoized, this._experimentProvider.experimentId);
```

Comment:
This is fine for here. I would assume we'll call `tabTracker.init` with appropriate arguments in the path that we don't initialize `ActivityStream`, so we'll want to do that in a followup issue. There is potential for oddness of passing around a partially-`init`-ed object as there's potential for accidentally multiply-`init`-ing. We could add some sanity checks, but probably not too important.